src/main/java/com/lob/client/AsyncLobClient.java

```diff
@@ -532,12 +532,16 @@ public void onThrowable(final Throwable t) {
         public T onCompleted(final Response response) throws Exception {
             if (isSuccess(response)) {
                 final T value = MAPPER.readValue(response.getResponseBody(), clazz);
```

Comment:
Is there are reason this needs to be a `T` and not an `AbstractResponse` as it's casted below?